app/styles/components/course-search-result.scss

```diff
@@ -0,0 +1,31 @@
+.course-search-result {
+  margin-bottom: 1rem;
+  max-width: 40rem;
+
+  .matched-in {
+    color: $darker-grey;
+    display: block;
+    font-size: .7rem;
+  }
+
+  .show-more {
+    font-size: .75rem;
+  }
+
+  .global-search-tags {
+    margin: .125rem 0;
+    overflow: hidden;
+
+    .global-search-tag {
+      background: lighten($grey, 5%);
+      border: 1px solid lighten($dark-grey, 10%);
+      border-radius: .125rem;
+      color: $darker-grey;
```

Comment:
There isn't enough contrast here to be legible. The background needs to be lighter or else the color should be darker.

Reply:
I will lighten the background a little more.

Reply:
Still not great, but I'm ok with it for now.